src/main/java/io/qdrant/client/QdrantGrpcClient.java

```diff
@@ -174,10 +177,26 @@ public static class Builder {
     }
 
     Builder(String host, int port, boolean useTransportLayerSecurity) {
-      this.channel = createChannel(host, port, useTransportLayerSecurity);
+      String clientVersion = getClientVersion();
+      String javaVersion = System.getProperty("java.version");
+      String userAgent = "java-client/" + clientVersion + " java/" + javaVersion;
+      this.channel = createChannel(host, port, useTransportLayerSecurity, userAgent);
       this.shutdownChannelOnClose = true;
     }
 
+    private String getClientVersion() {
+      String clientVersion = "Unknown";
+      try {
+        Manifest manifest =
+            new Manifest(Builder.class.getResourceAsStream("/META-INF/MANIFEST.MF"));
+        Attributes attributes = manifest.getMainAttributes();
+        clientVersion = attributes.getValue("X-Qdrant-Version");
```

Comment:
This reads from https://github.com/qdrant/java-client/blob/d71b260cd52173822bfc05d064f299692bd74162/build.gradle#L73

Which is the version of the proto files we've downloaded.
We should get `packageVersion` instead. https://github.com/qdrant/java-client/blob/d71b260cd52173822bfc05d064f299692bd74162/gradle.properties#L7-L8

Reply:
I'll extend manifest with X-Client-Version then.